FLEDGE.md

```diff
@@ -217,6 +219,16 @@ The logic in `scoreAd()` has access to the full auction configuration object, wh
 Note that `scoreAd()` does not have any way to _store_ information for use later on a different page.  In particular, if the ad scoring logic on day 1 observes a bid from a particular interest group, and then on day 2 the browser interest group membership expires, there is no way for the ad scoring logic on day 3 to "remember" the pre-expiration membership information.
 
 
+#### 2.4 Component Auctions
+
+Seller worklets running in component auctions behave a little differently.  They still expose a scoreAd() funcion to score each bid from the component auction's buyers, however all of its arguments come from the component auction, including `auctionConfig`. `browserSignals` has an additional `topLevelSeller` field, which contains the seller of the top-level auction. Instead of returning just a bid, `scoreAd()` returns an object with the following fields:
```

Comment:
Hmm, I wonder if we're going to need a `perBuyerSignals` sort of thing for the final auction to pass info to each component auction.  WDYT?

Reply:
I don't think that's needed - the subauctions can provide anything the top-level SSP wants to pass in to BidderWorklets, as well as the inner SSP.  In general, I think we should try not to add things unless there's no other way to reasonably do something.

Reply:
No, they can pass it in via the component auction's perBuyerSignals array.  They need to have an agreement on the format of this data, anyways, both with the bidder and the SSP running the component auction.

Reply:
perBuyerSignals provides data in the context of the page, where both SSPs may or may not be running scripts, to the worklet.  The SSPs can coordinate in the context of the page to provide all needed information to perBuyerSignals.  I don't think we need to provide additional mechanisms to pass data via the AuctionConfig to bidder worklets.

There are also a bunch of variations here - maybe the publisher wants to pass different data to the same buyers with different sellers in component auctions, maybe another SSP has multiple subauctions, and the top-level SSP wants different information to different sub auctions with the same seller.  Maybe it wants an 'auctionSignals' that is also sent to all buyers and sellers in component auctions.  Maybe it wants a 'sellerSignals' that's also sent to all sellers in component auctions (or maybe it was a 'perSellerSignals' that lets it specify data to each seller).

I think that since there's already a way to do this (coordinate on the page with nested SSPs to pass data via the component auction) that this isn't needed, and it blows up the size of the API, the number of needed tests, and the number of things that can break independently of each other.  Web platform APIs should try to be as simple as possible, and I just don't think all this stuff is needed.